docs/references/api/resource_embedding.rst

```diff
@@ -1196,6 +1206,40 @@ You can use this to get the columns of a join table in a many-to-many relationsh
      }
    ]
 
-.. note::
+.. _spread_to_many_embed:
 
-  The spread operator ``...`` is borrowed from the Javascript `spread syntax <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax>`_.
+Spread One-To-Many and Many-To-Many relationships
+-------------------------------------------------
+
+For these relationships, the spread embedded resource columns will be aggregated into arrays.
+
+.. code-block:: bash
+
+   # curl -g "http://localhost:3000/directors?select=first_name,...films(film_titles:title,film_years:year)&first_name=like.Quentin*"
+
+  curl --get "http://localhost:3000/directors" \
+    -d "select=first_name,...films(film_titles:title,film_years:year)" \
+    -d "first_name=like.Quentin*"
+
+.. code-block:: json
+
+   [
+     {
+       "first_name": "Quentin",
+       "film_titles": [
+         "Pulp Fiction",
+         "Reservoir Dogs"
+       ],
+       "film_years": [
+         1994,
+         1992
+       ]
+     }
+   ]
+
+The order of the values inside the resulting array is unspecified.
```

Comment:
> Would something like this be hard to do?

> For the spread, this could then move the ORDER BY into the aggregate function call. This would only allow to specify a single ORDER BY for multiple spread aggregates - which I consider a good thing, because this would ensure the array items still match between arrays.

I don't think it'd be hard to do. The syntax is OK, internally it would need to treat every order done inside a `to_many` spread as an `array_agg` order for every column, instead of a subquery one.

Right now, the order as you mentioned in your example *works*: it orders the subquery and the aggregated columns will be sorted. But it's not guaranteed to behave the same way for other more complex cases, [as mentioned in this convo](https://www.postgresql.org/message-id/flat/CAN19dycxr2f%2BGtKLR4X3OMepFzRGaA-uM6dNS6EqaNUEQTScww%40mail.gmail.com). So yes, I'll implement the order by in the aggregate here.